src/global/main.gd

```diff
@@ -1,34 +1,59 @@
 extends Node
 
+#Player values
 var coins = 0
 var level = 0
+var health = 3
+var playerPosition
+
+#Time
 var timeEclapsed = 0
 var framesEclapsed = 0
 var totalFramesEclapsed = 0
-var health = 3
-var in_range = false
-var player_dir = 0
-var aggro = false
+
+#World Health
 var worldHealth = 8
-var playerPosition
+
+#Mov, Mode detection
 var quicksand = false
 var ice = false
-var resetPlayer = false
+
+#Knockback detectors
 var knockback = false
-var death_height = [100, 100, 200, 100, 200, 270, 400, 100]
 var fireKnockback = false
+
+#Detects death
+var resetPlayer = false
+var death_height = [100, 100, 200, 100, 200, 270, 400, 100]
+var below_death_height = false
+
+#Leaderboard values
 var gameComplete = true
+var leaderboardOffer = false
 var suffixq
 var suffixes
+
+#Avalanche hit detector
 var snowHit = 0
-var leaderboardOffer = false
+
+#Player mov suspended detector
+var suspendMovement = false
+
+#No Idea/ -- Robbin Please explain --
```

Comment:
this just the code for the player going flash flash and immunity, don't worry abt it